src/content/guides/language/evolution.md

```diff
@@ -41,6 +41,33 @@ on the Dart language GitHub repo.
 
 ## Changes in each release
 
+### Dart 3.3
+_Released 15 February 2024_
+| [Dart 3.3 announcement](https://medium.com/dartlang/dart-3-3-325bf2bf6c13)
+
+Dart 3.3 added some enchancements to the language:
+
+* [Extension types][] are a new feature in Dart that allow zero-cost wrapping
+  of existing type. They are similar to wrapper classes and to extension methods,
```

Comment:
I'm not sure if this is best/most accurate the solution, but I think a word or two is missing here:

```suggestion
  of values with an existing type. They are similar to wrapper classes and to extension methods,
```

Reply:
I know there was a lot of discussion around what exactly is being wrapped, what the value is in these situations, etc. so I'm going to stick with just "wrapping _an_ (missing word) existing type", not because you're wording is wrong, I just can't remember if saying "values" in this context was cause for concern in other reviews...